app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -27,6 +28,14 @@ export function useCapitalProjectsLayer() {
   const { managingCode, capitalProjectId } = useParams();
   const [searchParams] = useSearchParams();
   const managingAgency = searchParams.get("managingAgency");
+  const commitmentsTotalMin = searchParams.get("commitmentsTotalMin");
+  const commitmentsTotalMax = searchParams.get("commitmentsTotalMax");
+  const min = commitmentsTotalMin
+    ? parseFloat(commitmentsTotalMin)
+    : -1000000000000;
```

Comment:
Note for myself - still want to investigate a better way of disabling a filter when there is no value for it but this is a good way of handling it for this filter for now.